presto-hive/src/main/java/io/prestosql/plugin/hive/procedure/SyncPartitionMetadataProcedure.java

```diff
@@ -94,18 +97,19 @@ public Procedure get()
                 ImmutableList.of(
                         new Argument("schema_name", VARCHAR),
                         new Argument("table_name", VARCHAR),
-                        new Argument("mode", VARCHAR)),
+                        new Argument("mode", VARCHAR),
+                        new Argument("case_insensitive", BOOLEAN, false, FALSE)),
```

Comment:
Make this `case_sensitive` since “insensitive” is a negative and using negatives for booleans can be confusing. This is my fault since I suggested a bad name when filing the issue.